docs/admin/node-conformance.md

```diff
@@ -9,54 +9,52 @@ assignees:
 
 ## Node Conformance Test
 
-*Node conformance test* is a test framework validating whether a node meets the
-minimum requirement of Kubernetes with a set of system verification and
-functionality test. A node which passes the tests is qualified to join a
-Kubernetes cluster.
+*Node conformance test* is a containerized test framework validating whether a
+node meets the minimum requirement of Kubernetes with a set of system
+verification and functionality test. A node which passes the test is qualified
+to join a Kubernetes cluster.
 
 ## Limitations
 
 There are following limitations in the current implementation of node
-conformance test. They'll be improved in future version.
+conformance test.
 
 * Node conformance test only supports Docker as the container runtime.
-* Node conformance test doesn't validate network related system configurations
-  and functionalities.
 
 ## Prerequisite
 
 Node conformance test is used to test whether a node is ready to join a
```

Comment:
"To run node conformance test, a node must satisfy the same prerequisites as a standard Kubernetes node. At a minimum, the node should have the following processes installed:"